src/main/java/org/broadinstitute/hellbender/utils/dataflow/BucketUtils.java

```diff
@@ -192,4 +191,35 @@ public static boolean fileExists(String path, PipelineOptions popts) {
         }
         return true;
     }
+
+    /**
+     * Create a new GCS options with a given api key.
+     * @param apiKey a valid GCS api key
+     */
+    public static GCSOptions getAuthenticatedGCSOptions(String apiKey) {
+        Utils.nonNull(apiKey);
+        GCSOptions options = PipelineOptionsFactory.as(GCSOptions.class);
+        options.setApiKey(apiKey);
+        return options;
+    }
+
+    /**
+     * Returns the file size of a file pointed to by a GCS/HDFS/local path
+     *
+     * @param path The URL to the file whose size to return
+     * @param popts PipelineOptions for GCS (if relevant; otherwise pass null)
+     * @return the file size in bytes
+     * @throws IOException
+     */
+    public static long fileSize(String path, PipelineOptions popts) throws IOException {
```

Comment:
Or should we just nix this fn?  Initially I thought we would need this but turned out not to be the case.  @droazen is this useful to leave in?